tubular/scripts/frontend_utils.py

```diff
@@ -127,6 +153,17 @@ def create_version_file(self):
         except IOError:
             self.FAIL(1, 'Could not write to version file for app {}.'.format(self.app_name))
 
+    def copy_js_config_file_to_app_root(self, app_config, app_name):
+        """ Creates a copy of env.config.js(x) file to the app root """
+        source = app_config['JS_CONFIG_FILEPATH']
```

Comment:
[suggestion] Should we only attempt to copy the `JS_CONFIG_FILEPATH` file if `JS_CONFIG_FILEPATH` exists? Otherwise, will this end up calling `self.FAIL` simply when an MFE does not define `JS_CONFIG_FILEPATH`?

We should ensure MFEs without `JS_CONFIG_FILEPATH` don't fail GoCD builds to be able to incrementally migrate MFEs.